docs/source/conf.py

```diff
@@ -68,7 +68,7 @@
     "doc_module": ("traja",),
     "reference_url": {
         "numpy": "http://docs.scipy.org/doc/numpy",
-        "scipy": "http://docs.scipy.org/doc/scipy/reference",
+        # "scipy": "http://docs.scipy.org/doc/scipy/reference",
```

Comment:
Are we meant to remove the scipy reference?

Reply:
Fixed in e456902.